cmd/up/up.go

```diff
@@ -352,14 +352,18 @@ func (up *upContext) activate(isRetry, autoDeploy, build bool) error {
 		return fmt.Errorf("couldn't activate your development container (%s): %s", up.Dev.Container, err.Error())
 	}
 
+	log.Success("Development container activated")
+
 	if err := up.forwards(ctx); err != nil {
-		return fmt.Errorf("couldn't forward traffic to your development container: %s", err.Error())
+		if err == errors.ErrSSHConnectError {
+			return up.checkOktetoStartError(ctx, "Failed to connect to your development container")
+		}
+		return fmt.Errorf("couldn't connect to your development container: %s", err.Error())
```

Comment:
is the output of `err.Error` useful? we can log just the text and throw the full error in a log.info

Reply:
I find it useful. It is how it was before and people usually report this error without needing to send a doctor file